test/benchmark/shimmer/instrumentation.bench.js

```diff
@@ -8,7 +8,7 @@
 const benchmark = require('../../lib/benchmark')
 const shimmer = require('../../../lib/shimmer')
 
-const suite = benchmark.createBenchmark({ name: 'shimmer wrapping', delay: 0.01 })
+const suite = benchmark.createBenchmark({ name: 'shimmer wrapping', runs: 100 })
```

Comment:
👍

Reply:
I'm not sure I follow how this solves the issue.  This just reduces the # of runs by 10x.  Isn't this telling us something isn't performant?

Reply:
Looking at the results, the slow test is `bootstrapInstrumentation`. I really don't understand why these are being benchmarked though, these aren't really in a hot path. This is called once at startup.